src/components/Resources/Card.tsx

```diff
@@ -0,0 +1,126 @@
+        >
+            {text}
+        </div>
+    )
+}
+
+/**
+ * This is the Card component rendered as a resource on the resources page.
+ *
+ * @param props - component props
+ * @param props.resource - a resource data object defined in resourceItems
+ */
+export const Card: FunctionComponent<Card> = ({ resource }) => {
+    // CTA text mapping
+    const watchNow = new Set(['virtual event', 'video'])
+    const readThe = new Set(['blog post', 'guide', 'customer story'])
+    let ctaText = 'Read more'
+    if (watchNow.has(resource.contentType)) {
+        ctaText = 'Watch now'
+    } else if (readThe.has(resource.contentType)) {
+        ctaText = `Read the ${resource.contentType === 'customer story' ? 'story' : resource.contentType}`
+    }
+
+    // Content Type background mapping
+    const backgrounds: BackgroundVariant = {
+        'virtual event': 'tw-from-vermillion-100 tw-to-violet-400',
+        video: 'tw-from-violet-100 tw-to-cerise-300',
+        'blog post': 'tw-from-violet-200 tw-to-vermillion-300',
+        guide: 'tw-from-blue-200 tw-to-violet-400',
+        'customer story': 'tw-from-green-100 tw-to-blue-300',
+    }
+
+    return (
+        <div className="tw-bg-white tw-shadow-md tw-rounded-lg md:tw-min-h-[469px]">
+            <div
+                className={classNames(
+                    'tw-h-[10%] tw-overflow-hidden tw-rounded-t-lg tw-flex tw-items-center tw-px-sm tw-bg-gradient-to-r',
+                    backgrounds[resource.contentType]
+                )}
+            >
+                <div className="first-letter:tw-capitalize tw-font-medium">{resource.contentType}</div>
+            </div>
+
+            {/* Card Info */}
+            <div className="tw-flex tw-flex-col tw-p-sm tw-h-[90%]">
+                <h4 className="tw-mb-sm">
+                    {resource.title.length > 100 ? `${resource.title.slice(0, 97)}...` : resource.title}
+                </h4>
+
+                <div className="flex-wrap tw-flex tw-mb-sm">
+                    {resource.subjects
+                        .slice(0, 3)
+                        .sort()
+                        .map(subject => (
+                            <FilterPill key={subject} text={subject} />
+                        ))}
+                    {!!resource.subjects.slice(3).length && (
+                        <div className="tw-group tw-relative">
+                            <FilterPill text={`+${resource.subjects.slice(3).length}`} />
+                            <div className="tw-hidden group-hover:tw-flex group-hover:tw-animate-fadeIn tw-absolute tw-left-0 tw-top-0 tw-shadow-md tw-bg-white tw-pt-xxs tw-pl-xxs tw-rounded-md tw-flex-col tw-cursor-default">
+                                {resource.subjects.slice(3).map(subject => (
+                                    <FilterPill key={subject} text={subject} light={true} />
+                                ))}
+                            </div>
+                        </div>
+                    )}
```

Comment:
Thanks! I pushed some updates that now use the Badge component for better code reuse, instead of this internal component (FilterPill) that I started with 😁 The Badge is now essentially an enhanced and flexible version of this.

Reply:
Right, I missed that, should've suggested it. Looks great! Love the badge